src/megaclient.cpp

```diff
@@ -13298,23 +13346,21 @@ bool MegaClient::syncdown(LocalNode* l, LocalPath& localpath, bool rubbish)
                 }
```

Comment:
We can't remove the ll->deleted test here - the lines above may have set it false since the last time we tested it.   Unfortunately our auto tests are not thorough enough yet to detect this error.

Reply:
My sincere apologies for the mistake, Matt. Great you checked my commits :sweat: